tasks/services/works.js

```diff
@@ -89,6 +94,112 @@ function createOrderPlans (hook) {
     .then(() => hook)
 }
 
+function sendOrderStartEmail (credential, appConfig, mailer, order) {
```

Comment:
recommend using a single `options` argument rather than many positional arguments, so:

```js
function sendOrderStartEmail ({ credential, appConfig, mailer, order }) {
```

or

```js
function sendOrderStartEmail (options) {
  const {
    credential,
    appConfig,
    mailer,
    order
  } = options
}
```

Reply:
refactor to `createStartOrderEmail`